### PR TITLE
Hook up tap devices to bridges

### PR DESCRIPTION
**Pull request by zenhack**
This is a bit more of a draft than the others. This adds a hook to the openvpn configs to attach the tap devices to the appropriate bridges, and adds an adaptation of the create-bridges script to the repo.

On the latter point, we may want to just keep using the HIL version, since that understand's HIL's allocation scheme.

After this, what's left is:

1. I want to do some proper integration testing of all this; I've been testing the stuff that touches the system manually with vagrant, but that won't scale in the long term.
2. We need to actually put auth(z) stuff in place.
3. Sensible HTTP server config stuff, probably shared with obmd.
4. Cohesive user-facing documentation.

...once that's all in place I think this is actually ready to go.

### CODE REVIEW

**Review by naved001 — APPROVED** [review]

File: cmd/hil-vpnd/main.go [hunk 1]
```diff
@@ -1,12 +1,41 @@
...
+	if err := env.Parse(&cfg); err != nil {
+		log.Fatal("Parsing config from environment: ", err)
+	}
+	if cfg.MinPort > cfg.MaxPort {
+		log.Fatal("Config error: MIN_VPN_PORT is greater than MAX_VPN_PORT")
+	}
+	if cfg.MinPort < 1024 {
+		log.Fatalf("MIN_VPN_PORT specifies a privileged port (%d)", cfg.MinPort)
+	}
+	if cfg.MaxPort >= (1 << 16) {
```
Comment (ianballou) [inline cmd/hil-vpnd/main.go]: is this `1 << 16` just for readability?

**Review by ianballou — APPROVED** [review]
Can't find anything wrong here! All good, just a quick question.

File: cmd/hil-vpnd/main.go (hunk 1 above)
Comment (zenhack) [reply cmd/hil-vpnd/main.go]: Yes.

**Comment by zenhack** [comment]
Rebased. will merge assuming travis passes.

**Comment by zenhack** [comment]
Merging.

**Pull request merged**